user-service/app/codegen/user_service_pb2_grpc.py

```diff
@@ -0,0 +1,82 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+import grpc
+
+import app.codegen.user_service_pb2 as user__service__pb2
```

Comment:
This part has been changed. Originally it was

```python
import user_service_pb2 as user__service__pb2
```

And that does't work since we're running the app from `server.py`.